zeppelin-web/src/components/baseUrl/baseUrl.service.js

```diff
@@ -35,6 +35,11 @@ angular.module('zeppelinWebApp').service('baseUrlSrv', function() {
     return wsProtocol + '//' + location.hostname + ':' + this.getPort() + skipTrailingSlash(location.pathname) + '/ws';
   };
 
+  this.getJobManagerWebsocketUrl = function() {
+    var wsProtocol = location.protocol === 'https:' ? 'wss:' : 'ws:';
+    return wsProtocol + '//' + location.hostname + ':' + this.getPort() + skipTrailingSlash(location.pathname) + '/ws/job';
```

Comment:
This looks very similar to the implementation of `getWebsocketUrl`.

As @Leemoonsoo pointed before, copy-paste and duplication reduces code maintainability, so is there a particular reason why we can not user

```javascript
  return getWebsocketUrl() + '/job';
```
?